### PR TITLE
replace `else: if:` with a literal `elif`

### PR DESCRIPTION
**Pull request by Hexiro**
In my opinion, `elif`s look better because there is no change in indentation level, but these should function the exact same. I'm not experienced with python 2, but I can't imagine this would make a difference.

### CODE REVIEW

File: steam/client/__init__.py [hunk 1]
```diff
@@ -297,9 +297,8 @@ def send(self, message, body_params=None):
         """
         if not self.connected:
             self._LOG.debug("Trying to send message when not connected. (discarded)")
-        else:
-            if body_params and isinstance(message, MsgProto):
-                proto_fill_from_dict(message.body, body_params)
+        elif body_params and isinstance(message, MsgProto):
+            proto_fill_from_dict(message.body, body_params)
 
             CMClient.send(self, message)
```
Comment (Nothing4You) [inline steam/client/__init__.py:303]: this actually changes the logic as `CMClient.send(self, message)` would only be called when the if condition matches, while the original version ran it in both cases in the else block

Comment (Hexiro) [reply steam/client/__init__.py:303]: gotcha. From my brief look it looks like you're right. Next time I'm at my computer i'll take a closer like and most likely revert that part.

Comment (Hexiro) [reply steam/client/__init__.py]: thanks for calling me out on that one mate. I've reverted that change now.

**Review by rossengeorgiev — APPROVED** [review]
LGTM

**Pull request merged**